dev/com.ibm.ws.microprofile.faulttolerance.spi/src/com/ibm/ws/microprofile/faulttolerance/spi/ExecutorBuilder.java

```diff
@@ -24,6 +24,8 @@
 
     public ExecutorBuilder<T, R> setTimeoutPolicy(TimeoutPolicy timeout);
 
+    public ExecutorBuilder<T, R> setMetricRecorder(MetricRecorder metricRecorder);
```

Comment:
javadoc

Reply:
This whole class needs javadoc, raised #3515 to do it in another PR.